owl/EASE-SAY.owl

```diff
@@ -63,6 +111,33 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Instrument -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Instrument"/>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Patient -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Patient"/>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#AgentRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#AgentRole">
```

Comment:
It is very common to speak of "agents" in other contexts, e.g. "cleaning agent".
Would they be included with this role definition?